lib/getPostcssResult.js

```diff
@@ -94,8 +94,16 @@ function getCustomSyntax(customSyntax) {
 		try {
 			resolved = require(customSyntax);
 		} catch (error) {
-			// @ts-expect-error -- TS2571: Object is of type 'unknown'.
-			if (error && typeof error === 'object' && error.code === 'MODULE_NOT_FOUND') {
+			if (
+				error &&
+				typeof error === 'object' &&
+				// @ts-expect-error -- TS2571: Object is of type 'unknown'.
+				error.code === 'MODULE_NOT_FOUND' &&
+				// @ts-expect-error -- TS2571: Object is of type 'unknown'.
+				error.message.includes(customSyntax)
+			) {
+				console.error('Stylelint MODULE_NOT_FOUND', error);
+
 				throw new Error(
 					`Cannot resolve custom syntax module "${customSyntax}". Check that module "${customSyntax}" is available and spelled correctly.`,
```

Comment:
[suggestion] How about embedding a caused error into the existing message like this? (instead of `console.error()`)

```suggestion
					`Cannot resolve custom syntax module "${customSyntax}". Check that module "${customSyntax}" is available and spelled correctly.\n\nCaused by: ${error}`,
```

---

BTW, `Error.prototype.cause` is available since ES2021 and Node 16.9.0 and seems suitable for this case. But unfortunately, we need to support older Node.js versions for a while. 😓 

See also:
- https://2ality.com/2021/06/error-cause.html
- https://github.com/tc39/proposal-error-cause
- https://nodejs.org/en/blog/release/v16.9.0/#error-cause

Reply:
TIL `Error.prototype.cause`  :+1: